server/sonar-web/src/main/js/apps/create/components/OrganizationKeyInput.tsx

```diff
@@ -61,7 +61,7 @@ export default class OrganizationKeyInput extends React.PureComponent<Props, Sta
     this.setState({ validating: true });
     return getOrganization(key)
       .then(organization => {
-        if (this.mounted) {
+        if (this.mounted && this.state.value === key) {
```

Comment:
This checkFreeKey() is a debounce function in this component. 
When clicking backspace with key-up, The last call of this function is made last character instead of empty value because empty is not a valid org key. This results in mismatch of value of key in props and state variable. 
Checking equality of key with state variable fixes the issue.